tests/test_external_roundtrip.sh

```diff
@@ -0,0 +1,6 @@
+# This test ensures that we can call encode | decode as two distinct processes.
+
+echo "external_roundtrip: Baseline dictionary"
+echo 'function f() {}' | target/debug/binjs_encode advanced entropy | target/debug/binjs_decode advanced entropy
```

Comment:
Let's maybe check the output of roundtrip as well? (just for basic sanity check)

Reply:
Yeah, I'm rewriting this to check it.